.github/workflows/ci.yaml

```diff
@@ -14,8 +14,8 @@ on:
 env:
   ## Some version numbers that are used during CI
   ormolu_version: 0.7.2.0
-  jit_version: "@unison/internal/releases/0.0.20"
-  runtime_tests_version: "@unison/runtime-tests/releases/0.0.1"
+  jit_version: "@unison/internal/releases/0.0.21"
```

Comment:
just curious why did the jit version change?

Reply:
I implemented the new serialization format for the jit. This means it now has access to the cacheability information as well.

I was going to implement the top level value behavior in the jit, too, but after some struggling, I realized that my `stronglyConnectedComponents` implementation does not actually output a topological sorting of the SCCs, which I need to output the racket definitions in an acceptable order. So I'll probably just merge this as is.

Reply:
I guess I wouldn't have expected it to change in an interpreter-related PR, but is it just a compatibility thing? That the interpreter was updated to match the new serialization format?

Reply:
The interpreter, jit and ucm use a common format for code interchange, so when that changes (to include the cacheability information), the jit needs to be updated to understand it. Otherwise ucm could send it stuff that would just make it bomb. For instance, the format is used for `run.native` and `compile.native` to tell the executable what code to run/compile.